src/include/wiredtiger.in

```diff
@@ -49,12 +49,6 @@ extern "C" {
 #define	WT_ATTRIBUTE_LIBRARY_VISIBLE	__attribute__((visibility("default")))
 #endif
 
-#ifdef SWIG
```

Comment:
@keithbostic, @agorrod, that's correct.  The SWIG interface code now causes `wiredtiger.h` to be included at the top of the wrapper, so we can remove the include from `wiredtiger.in`